programs/gzip.c

```diff
@@ -583,6 +589,15 @@ tmain(int argc, tchar *argv[])
 				return 1;
 			}
 			break;
+		case 't':
+			options.test = true;
+			options.decompress = true;
+			options.to_stdout = true;
+			/*
+			 *  test utilises to_stdout (-c) option
+			 *  except that it doesn't write anything out
```

Comment:
This doesn't really explain why `to_stdout` is set.  I recommend using the comment that I suggested.

Reply:
You're right, I meant `libdeflate_gzip_decompress_ex()`, which returns `result`.